packages/ListBox/src/ListBox.js

```diff
@@ -12,37 +12,44 @@ import Popover from "./components/Popover";
 import Trigger from "./components/Trigger";
 import useListBox from "./useListBox";
 import { OnChangeContext } from "./store/OnChangeProvider";
+import { PropsContext } from "./store/PropsProvider";
 import handleImperative from "./imperative";
-
 import {
   useAdjustWidth,
   useChildrenChange,
   useHasFooter,
-  useIsDisabled,
   useIsPopOverOpen,
   useOnScrolled,
   useOptionSelected,
 } from "./hooks";
 
 export function ListBox(props) {
-  const [state] = useListBox();
   const {
     children,
     hasImplicitAll,
     height,
-    placeholder,
-    trigger: _trigger,
-    footer,
-    filter,
+    isOpen,
+
     box = { props: {} },
+    filter,
+    footer,
+    trigger: _trigger,
+
+    ...moreProps
   } = props;
+
   const I18n = useI18n();
+  const [state] = useListBox();
+  const { isInline, isReadOnly } = React.useContext(PropsContext);
+
+  const hasOptions = Boolean(React.Children.count(children));
+
   const propsForTrigger = {
     hasClearButton: true,
     hasImplicitAll,
     onClickClear: null,
-    placeholder: placeholder || I18n.t("listBox.trigger.placeholder"),
-    onFooterClickAccept: footer ? footer.props.onClickAccept : null,
+    onClickFooterAccept: footer ? footer.props.onClickAccept : null,
+    morePropsForTrigger: moreProps,
```

Comment:
Since the consumer can add more `moreProps` directly to `<ListBox.Trigger>` which should go on the container element, not the underlying `<RawButton>`, these `moreProps` from `<ListBox>` need to be separated.